### ISSUE TITLE
Yarn Build command fails

### ISSUE THREAD

**Issue by kcelestinomaria** [scale-it/algo-builder#330]
Yarn build command fails at the @algo-builder/algob process. It looks like you changed some dependencies and did not update the build process. Here is the error message: `node_modules/@algo-builder/runtime/build/logicsig.d.ts:3:34 - error TS2305: Module '"algosdk"' has no exported member 'MultiSigAccount'.

3 import { LogicSigBase, MultiSig, MultiSigAccount } from "algosdk";
                                   ~~~~~~~~~~~~~~~

src/lib/algo-operator.ts:310:36 - error TS2339: Property 'DeploySSC' does not exist on type 'typeof TransactionType'.

310       type: rtypes.TransactionType.DeploySSC,
                                       ~~~~~~~~~

src/lib/algo-operator.ts:313:7 - error TS2322: Type '{ type: any; sign: rtypes.SignType.SecretKey; fromAccount: algosdk.Account; approvalProgram: string; clearProgram: string; approvalProg: Uint8Array; clearProg: Uint8Array; ... 10 more ...; lease: Uint8Array | undefined; }' is not assignable to type 'ExecParams'.
  Object literal may only specify known properties, and 'approvalProgram' does not exist in type 'ExecParams'.

313       approvalProgram: approvalProgram,
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/lib/algo-operator.ts:366:36 - error TS2339: Property 'OptInSSC' does not exist on type 'typeof TransactionType'.

366       type: rtypes.TransactionType.OptInSSC,
                                       ~~~~~~~~` @robert-zaremba @Invertisment @sebastiangula @amityadav0 @as04

**Comment by amityadav0** [comment]
@kcelestinomaria Thanks for catching. I have added a PR for this issue.

**Comment by robert-zaremba** [comment]
We just released `1.0.1` with the fix. Thanks for reporting.
